base/interactiveutil.jl

```diff
@@ -49,6 +49,8 @@ function edit(path::AbstractString, line::Integer=0)
         cmd = line != 0 ? `$command $path -l $line` : `$command $path`
     elseif startswith(name, "subl") || startswith(name, "atom")
         cmd = line != 0 ? `$command $path:$line` : `$command $path`
+    elseif startswith(name, "code")
```

Comment:
That seems reasonable, but I would be careful with the capitalization on Windows, since Windows is case insensitive. Perhaps for the sake of catching exact matches, you could do something like
```julia
elseif name == "code" || (is_windows() && uppercase(name) == "CODE.EXE")
```
?